README.md

```diff
@@ -85,6 +87,6 @@ Your output could be something like:
 The most common mood is :-)
 ```
 
-**think**: Should you use an array or a hash to solve this problem? Why?
+**think**: Should you use an array or a hash to solve this problem? Why? I used a hash because order doesn't matter (I think)- I'm concerned about being able to tie the count to a specific emoji and not a specific position.
 
-**think**: What if we eventually want to add feelings to our analysis? Can we write this code in a way that will prevent us from having to re-write it later?
+**think**: What if we eventually want to add feelings to our analysis? Can we write this code in a way that will prevent us from having to re-write it later? We could have a EMOJI_MOOD constant that store the possible mood emojis in an array and then use that to match them.  Right now I'm thinking we would loop through the array and compare the strings, but that would involved a lot of loops for each case so there's probably a more elegant way to do that!  In that case I would choose an array, because although order doesn't matter, I'm not trying to tie things together, either.  I just want to store all the possibilities.
```

Comment:
This is where a hash becomes really nice, because the keys can be generated on the fly for counting. And we have the associated emoticon with each count without having to either keep track ourselves or have lots of counters. I'll show my code in class and maybe this will make more sense.